cli.js

```diff
@@ -1,13 +1,26 @@
 #!/usr/bin/env node
 
 var chalk = require("chalk");
-var lib = require(".");
+var argv = require("yargs").argv;
```

Comment:
I actually wanted to refactor lerna's (well, at the time, asini's) usage of meow into yargs, it's got so many neat bells and whistles (with better help out of the box).